thunderfish/FishTracker.py

```diff
@@ -302,6 +315,7 @@ def bw_psd_and_eod_plot(self, power, freqs, bwin, win_width, data, psd_type, fis
             # get time for plot
             plot_time = ((np.arange(len(mean_pulse_data)) * 1.0 / self.rate) - eod_plot_tw / 2) * 1000  # s to ms
 
+
```

Comment:
PEP says only one empty line to separate blocks in code ...

as I said I'm in the nit-picking mood :)